addon/utils/utils.js

```diff
@@ -4,9 +4,14 @@
  */
 
 import Ember from 'ember';
+import require from 'require';
 
 export function hasEmberData() {
-  return typeof self.DS !== 'undefined';
+  return require.has('ember-data');
```

Comment:
Weird, this has been tested on Ember 1.11 to canary. What version of ember-cli are you on?

/cc @stefanpenner